conference/forms.py

```diff
@@ -74,7 +74,7 @@ def _media(self):
         )
     media = property(_media)
 
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
```

Comment:
i assume it's for some inheritance/API compatibility, that said, renderer is passed as argument but not used in the function.

Reply:
Exactly, it's for compatibility with the new signature of the `render` method.